graph/telemetry/istio/appender/appender.go

```diff
@@ -30,6 +30,8 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 				requestedAppenders[AggregateNodeAppenderName] = true
 			case DeadNodeAppenderName:
 				requestedAppenders[DeadNodeAppenderName] = true
+			case HealthAppenderName:
+				requestedAppenders[HealthAppenderName] = true
```

Comment:
Again, here ther is overlap with Edgar and https://github.com/israel-hdez/swscore/pull/23.  I think you can leverage the new, consolidate health finalizer-appender.